server/service/src/pack_variant/query.rs

```diff
@@ -19,7 +19,8 @@ pub fn get_pack_variants(ctx: &ServiceContext) -> Result<Vec<ItemPackVariant>, R
     let stock_on_hand = StockOnHandRepository::new(connection).query(Some(
         StockOnHandFilter::new().store_id(EqualFilter::equal_to(store_id)),
     ))?;
-    let pack_variants = PackVariantRowRepository::new(connection).load_all()?;
+    let pack_variants = PackVariantRepository::new(connection)
```

Comment:
Filtering out is_active in the backend since frontend didn't seem to like this... this means we'll need another endpoint to return deleted pack variants when implementing #2530

Reply:
Great idea (: